packages/web/src/components/withdraw-usdc-modal/components/TransferSuccessful.tsx

```diff
@@ -57,11 +65,11 @@ export const TransferSuccessful = () => {
       <div className={styles.destination}>
         <TextRow left={messages.destinationAddress} />
         <Text variant='body' size='medium' strength='default'>
-          {wallet}
+          {addressValue}
         </Text>
         <HarmonyPlainButton
           style={{ padding: 0 }}
-          onClick={() => openExplorer(signature)}
+          onClick={() => openExplorer(signature ?? '')}
```

Comment:
should we just hide this button if the sig doesn't exist for whatever reason?

Reply:
I think we probably need an additional error state here. Will confer w/ design. Let's see how this comes together when we wire up the signature bits to the sagas.